zp-relayer/workers/poolTxWorker.ts

```diff
@@ -33,27 +33,28 @@ export async function createPoolTxWorker<T extends EstimationType>(
   await updateNonce(nonce)
 
   const CHAIN_ID = await getChainId(web3)
-  const poolTxWorkerProcessor = async (job: Job<TxPayload[]>) => {
+  const poolTxWorkerProcessor = async (job: Job<BatchTx, PoolTxResult[]>) => {
     const sentTxNum = await sentTxQueue.count()
     if (sentTxNum >= config.maxSentQueueSize) {
       throw new Error('Optimistic state overflow')
     }
 
-    const txs = job.data
+    const txs = job.data.transactions
+    const traceId = job.data.traceId
 
-    const logPrefix = `POOL WORKER: Job ${job.id}:`
-    logger.info('%s processing...', logPrefix)
-    logger.info('Received %s txs', txs.length)
+    const jobLogger = scopedLogger(`POOL WORKER: Job ${job.id}: `, { traceId })
```

Comment:
I personally would prefer to have all the fields inside of the property object:
```
const workerLogger = logger.child({ worker: 'pool' })
...
const jobLogger = workerLogger.child({ jobId: job.id, traceId })
```
Wdyt?

Reply:
Agree, looks more concise. I will update this.